README.md

```diff
@@ -0,0 +1,8 @@
+The signtool is an utility that can be used to issue sign/verify requests for a development or
```

Comment:
The op-signtool name seems to be something specific for openpower. IIRC, signtool tool will be used for openpower and powervm.